### DISCUSSION TITLE
Using anaconda with LIMA

### DISCUSSION THREAD

**Discussion by Micbut** [lima-vm/lima#1334] [Q&A]
Hi, 
I would like to use linux on my macbook m1 to run this software: https://plantscan3d.readthedocs.io/en/latest/userguide/installation.html

The reason is that it doesn't work with apple m1.

I successfully downloaded LIMA but I don't know how to proceed. 
Is it possible to use anaconda with LIMA to install Plantscan3d?
Is there a way to open a linux desktop with Lima? So that I can interact with a graphical user Interface

I hope my question is clear
thank you

**Comment by afbjorklund** [comment]
There is no GUI example in the default templates, but it could be possible to add one.

* https://github.com/lima-vm/lima/discussions/987

> The reason is that it doesn't work with apple m1.

Running in emulation would be slow, so it would be better to find an ARM version of it...

* https://docs.anaconda.com/anaconda/install/graviton2/

Not sure if the installer works with the "ubuntu" template,  but you could try it perhaps.

Otherwise it would probably be better to ask anaconda (or planscan3d) for install help ?
